reports/reporting/__init__.py

```diff
@@ -47,3 +46,25 @@ def get_report_per_study(re_id, sd_id):
               f' for release {re_id} and study id {sd_id}')
 
     return jsonify(resp['Item']), 200
+
+
+@reports_api.route('/<sd_id>/state=<state>', methods=['GET'])
```

Comment:
Would a better route be `/<sd_id>?state=<state>`?
Filters are typically url arguments.

Reply:
ya the problem is we are using the same route `/reports/<id>` for releases and studies and both have different ways to get the data. i am thinking in future we probably make a single  `reports/` endpoint and make `release_id` and `study_id`  and other fields filterable.